scipy/stats/tests/test_distributions.py

```diff
@@ -1496,6 +1512,13 @@ def test_fit_warnings(self):
         assert_raises(FitDataError, stats.pareto.fit, [5, 2, 3], floc=1,
                       fscale=3)
 
+    def test_negative_data(self, rng):
+        # The purpose of this test is to make sure that no warnings are raised
+        # for all negative data.
+        data = stats.pareto.rvs(loc=-130, b=1, size=100, random_state=rng)
+        assert np.all(data < 0)
+        _ = stats.pareto.fit(data)
```

Comment:
Is this a leftover? What is this testing?

Reply:
When `floc` is free, `root_scalar` is used to find the scale. The initial guess for the scale was originally `np.max(data)`, but that didn't work for all negative data (warnings raised / failure). This test addresses that change, though I am open to removing it since it seems to work. I added this test because the all negative data issue did not come up until I specifically looked for all negative data.

Reply:
Sorry GH is not helping here although I flagged just a single line. I am talking about the last line, since there is no assert or test on ` _ = stats.pareto.fit(data)
`

Reply:
The idea was to ensure that no warnings are raised due to improperly set brackets. A runtimewarning is raised when using a bracket of `np.max(data)`, which I tried previously. Now it is using `np.ptp(data)` and no runtimewarnign is raised. 

I see it could be confusing that there is no assert, I'm not sure how to assert that no runtimewarnings were raised.

Reply:
Oh ok, then if this cannot be tested while you do another call, then can you add a comment so that we don't remove it down the road?